internal/wait/wait.go

```diff
@@ -41,6 +42,34 @@ func ClusterPolicyReady(apiClient *clients.Settings, clusterPolicyName string, p
 		})
 }
 
+// NicClusterPolicyReady Waits until nicClusterPolicy is Ready.
+func NicClusterPolicyReady(apiClient *clients.Settings, nicClusterPolicyName string, pollInterval,
+	timeout time.Duration) error {
+	return wait.PollUntilContextTimeout(
+		context.TODO(), pollInterval, timeout, true, func(ctx context.Context) (bool, error) {
+			nicClusterPolicy, err := nvidianetwork.Pull(apiClient, nicClusterPolicyName)
+
+			if err != nil {
+				glog.V(gpuparams.GpuLogLevel).Infof("ClusterPolicy pull from cluster error: %s\n", err)
+
+				return false, err
+			}
+
+			if nicClusterPolicy.Object.Status.State == "ready" {
+				glog.V(gpuparams.GpuLogLevel).Infof("NicClusterPolicy %s in now in %s state",
```

Comment:
IIUC it should use the network operator log level. Or an operator-agnostic one if you want them to be the same for all operators.

Reply:
I agree may be just use LogLevel for both, but will have to update GPU code in next PR.